docs/entity_air_pollutants.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Air Pollutants Entity
+sidebar_label: Air Pollutants
+---
+
+## Properties
+
+> Properties should always only return information from memory and not do I/O (like network requests). Implement `update()` or `async_update()` to fetch data.
+
+| Name | Type | Default | Description
+| ---- | ---- | ------- | -----------
+| state | string | **Required** | The current PM 2.5 level.
```

Comment:
State should not be part of the documented properties. The entity component should just fix that.

Reply:
Ok. Looks like we need to fix that in the `weather` documentation too as this was the source.